netlogo-gui/src/main/ide/CodeCompletionPopup.scala

```diff
@@ -5,13 +5,15 @@ package org.nlogo.ide
 import java.awt.{Color, Component, Dimension}
 import java.awt.event._
 import javax.swing._
+import javax.swing.event.DocumentEvent
 
 import org.nlogo.api.CompilerServices
-import org.nlogo.core.DefaultTokenMapper
+import org.nlogo.core.{DefaultTokenMapper, Token}
 import org.nlogo.window.SyntaxColors
 import org.nlogo.editor.EditorArea
+import org.nlogo.lex.Tokenizer
```

Comment:
This will need to be removed to get `depend` passing. Use `org.nlogo.core.TokenizerInterface` instead.